src/components/playground/result/progress.tsx

```
@@ -1,11 +1,42 @@
+  const { progress } = useScraper();
+
+  if (!progress) {
+    return null;
+  }
 
-  return <Progress value={progress} />;
+  return (
+    <div className="flex flex-col gap-2">
+      {steps.map(({ title, key }) =>
+        progress[key as ProgressKey] ? (
+          <div className="flex flex-col gap-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)